.buildkite/release_scenario/new-hotfix-release.yml

```diff
@@ -0,0 +1,24 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/a8c-ci-toolkit#3.0.1
+    - automattic/git-s3-cache#1.1.4:
+        bucket: "a8c-repo-mirrors"
+        # This is not necessarily the actual name of the repo or the GitHub organization
+        # it belongs to. It is the key used in the bucket, which is set based on
+        # the Buildkite pipeline slug
+        repo: "automattic/woocommerce-ios/"
+  # Common environment values to use with the `env` key.
+  - &common_env
+    # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
+    IMAGE_ID: xcode-14.3.1
+
+steps:
+  - label: "Start Code Freeze"
+    plugins: *common_plugins
+    env: *common_env
+    command: |
+      .buildkite/commands/configure-environment.sh
+
+      bundle exec fastlane new_hotfix_release version:$VERSION skip_confirm:true
```

Comment:
@hannahtinkler For the `new_hotfix_release` lane, we pass in the version number of the hotfix. I wasn't sure if `$VERSION` was the correct way to add that here when this will be called from the PHP config. Is there another way this command should be formatted?